README.md

```diff
@@ -16,41 +20,46 @@ For tutorials, samples, quick starts and other documentation, visit [Azure for J
 
 These libraries can be easily identified by their folder, package, and namespaces names starting with `azure-`, e.g. `azure-keyvault`.
 
-The libraries released in the November 2019 GA release:
-- [Identity](sdk/identity/azure-identity/README.md)
-- [Key Vault Keys](sdk/keyvault/azure-security-keyvault-keys/README.md)
-- [Key Vault Secrets](sdk/keyvault/azure-security-keyvault-secrets/README.md)
-- [Storage Blobs](sdk/storage/azure-storage-blob/README.md)
-- [Storage Blobs Batch](sdk/storage/azure-storage-blob-batch/README.md)
-- [Storage Blobs Cryptography](sdk/storage/azure-storage-blob-cryptography/README.md)
-- [Storage Queues](sdk/storage/azure-storage-queue/README.md)
+The libraries released in the GA November 2019 release:
+- [Identity](https://github.com/Azure/azure-sdk-for-java/blob/azure-identity_1.0.0/sdk/identity/azure-identity/README.md)
+- [Key Vault Keys](https://github.com/Azure/azure-sdk-for-java/blob/azure-security-keyvault-keys_4.0.0/sdk/keyvault/azure-security-keyvault-keys/README.md)
+- [Key Vault Secrets](https://github.com/Azure/azure-sdk-for-java/blob/azure-security-keyvault-secrets_4.0.0/sdk/keyvault/azure-security-keyvault-keys/README.md)
+- [Storage Blobs](https://github.com/Azure/azure-sdk-for-java/blob/azure-storage-blob_12.0.0/sdk/storage/azure-storage-blob/README.md)
+- [Storage Blobs Batch](https://github.com/Azure/azure-sdk-for-java/blob/azure-storage-blob_12.0.0/sdk/storage/azure-storage-blob-batch/README.md)
+- [Storage Blobs Cryptography](https://github.com/Azure/azure-sdk-for-java/blob/azure-storage-blob_12.0.0/sdk/storage/azure-storage-blob-cryptography/README.md)
+- [Storage Queues](https://github.com/Azure/azure-sdk-for-java/blob/azure-storage-blob_12.0.0/sdk/storage/azure-storage-queue/README.md)
 
 The libraries released in the November 2019 preview:
-- [App Configuration](sdk/appconfiguration/azure-data-appconfiguration/README.md)
-- [Event Hubs](sdk/eventhubs/azure-messaging-eventhubs/README.md)
-- [Event Hubs Checkpoint Store](sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-blob/README.md)
-- [Storage File Shares](sdk/storage/azure-storage-file-share/README.md)
-- [Key Vault Certificates](sdk/keyvault/azure-security-keyvault-certificates/README.md)
-- [OpenCensus Tracing](sdk/core/azure-core-tracing-opencensus/README.md)
+- [App Configuration](https://github.com/Azure/azure-sdk-for-java/blob/074dd7a33f58ec02f84bfb6b94fa48529a6997ad/sdk/appconfiguration/azure-data-appconfiguration/README.md)
```

Comment:
this doesn't look like the correct tag

Reply:
This is the correct commit. For some reason, no tags were created for App Config. Thus we are using comment hash .